support-frontend/assets/helpers/productPrice/__tests__/paperSavingsVsRetailTests.ts

```diff
@@ -96,8 +96,8 @@ const productPrices = {
 			EverydayPlus: {
 				Monthly: {
 					GBP: {
-						price: 29.2,
-						savingVsRetail: 41,
+						price: 47.62,
```

Comment:
Just checking, presume this is supposed to differ to 55.69 in `ProductInfoMock.ts`